ods_ci/tests/Resources/CLI/ModelRegistry/MRMS_TEST_RUNNER.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Most (all?) of the jupyter notebooks we use are in this repo: https://github.com/redhat-rhods-qe/ods-ci-notebooks-main - perhaps it'd be better to put this there as well and clone it into your Notebook instead of uploading it from local?
cc @jgarciao @jstourac you might have an opinion on this as well?